src/collectionData.js

```diff
@@ -0,0 +1,109 @@
+export default class TodoListData {
+  constructor() {
+    this.todoListTasks = JSON.parse(window.localStorage.getItem('todoList')) || [];
+    this.taskRemoveButtons = [];
+  }
+
+  addTask(task) {
+    task.taskIndex = this.todoListTasks.length;
+    this.todoListTasks.push(task);
+    this.updateStorage();
+  }
```

Comment:
- [ ] Kindly make sure always the `index` of the task starts at `1`. That means if we added the second task make sure its `index` is `2` instead of being `1`. Kindly implement that to meet the project requirement.
- [OPTIONAL] Kindy make sure to clear the `input` once we added the new task for achieving a better user experience.

Reply:
thanks for the suggestion will work on it

Reply:
fixed